tests/test_sample_rayjob_yamls.py

```diff
@@ -33,6 +33,12 @@
                         {'path': filepath, 'name': filename, 'cr': k8s_object}
                     )
                     break
+
+    skip_tests = {
+        'ray-job.tpu-v6e-singlehost.yaml': 'Skip this test because it requires TPU resources.',
```

Comment:
Do you need this check here? The list of YAML files is in line 24

Reply:
Oh oops I missed this line, I thought it was getting all the yamls like the RayCluster test. Removed in [8f56179](https://github.com/ray-project/kuberay/pull/2445/commits/8f56179b1e9f4cc22343c27d549747c3d70efe9b).